core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileUtil.java

```diff
@@ -93,4 +91,13 @@ public static int getGenePanelId(String panelId) {
         GenePanel genePanel = genePanelRepositoryLegacy.getGenePanelByStableId(panelId).get(0);
         return genePanel.getInternalId();
     }
+
+    public static boolean geneInPanel(CanonicalGene gene, GenePanel genePanel) {
```

Comment:
I know this is after the fact, but one thought I had while reading this is that we seem to be adding new dependencies into the core module here.  Maybe this functionality should be in a util class in another module?
Also, I'm surprised that we need to loop through the genes in a panel in order to find if a gene is a member. We could put this functionality into class GenePanel ... and store (if necessary) a hash set of entrezGeneId for members.